pyccel/ast/operators.py

```diff
@@ -938,7 +938,7 @@ def _calculate_type(cls, arg1, arg2):
         elif len(possible_class_types) == 1:
             class_type = possible_class_types.pop().switch_basic_type(dtype)
         else:
-            description = f"({arg1} {cls.op} {repr(arg2)})" # pylint: disable=no-member
+            description = f"({repr(arg1)} {cls.op} {repr(arg2)})" # pylint: disable=no-member
```

Comment:
This also works, and is more compact:
```suggestion
            description = f"({arg1!r)} {cls.op} {arg2!r})" # pylint: disable=no-member
```

Reply:
https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals

Reply:
Ah nice! I didn't know about this sytax